docs/resources/server/recipes/index.md

```diff
@@ -148,6 +148,41 @@ The `ShapedRecipePattern` class, responsible for holding the in-memory represent
 `ShapedRecipePattern#setCraftingSize` is not thread-safe. It must be wrapped in an `event#enqueueWork` call.
 :::
 
+### Client-Side Recipes
+
+By default, Vanilla does not send any recipes to the [logical client][logicalside]. Instead, the `RecipeDisplay` is synced when a recipe is unlocked in the recipe book, and the `RecipePropertySet` / `SelectableRecipe.SingleInputSet` for proper behavior during interactions. However, these use cases are limited in scope, especially when more data is needed from the recipe itself. In these instances, NeoForge provides a may to send the full recipes for a given `RecipeType` to the client.
```

Comment:
"Instead, the `RecipeDisplay` is synced when a recipe is unlocked in the recipe book, and the `RecipePropertySet` / `SelectableRecipe.SingleInputSet` for proper behavior during interactions."
This sentence feels incomplete.